osx/build

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash -xe
+
+set -o pipefail
+
+brew install gtk+3
```

Comment:
I suggest to use only commands to _build_ grisbi in ``osx/build``

``brew install`` commands should be lkisted in ``INSTALL.MacOSX`` since they are only run once.